src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -600,7 +603,7 @@ public void PublishRelease_does_not_override_Configuration_property_across_forma
                });
 
             new DotnetPublishCommand(Log, helloWorldAsset.TestRoot)
-                .Execute(configOpt)
+                .Execute(configOpt,"-bl:C:\\users\\noahgilson\\it_passes_but_fails.binlog")
```

Comment:
Not sure if you intended this binlog to get emitted as well

Reply:
Definitely not, thanks! I meant to force push this earlier and remove it ...